features/specs/S-en-T-test-omgekeerd-ouders/Tg033.feature

```diff
@@ -362,6 +363,13 @@ Functionaliteit: Tg033 - Paassen-Pauw - Twee gehuwde vrouwen met 4 kinderen
     En heeft 'gezag' een 'ouder' met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 000000085 |
+    En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde           |
+      | type                             | GezamenlijkGezag |
+      | minderjarige.burgerservicenummer | 000000048        |
+      | ouder.burgerservicenummer        | 000000012        |
+      | derde.burgerservicenummer        | 000000024        |
```

Comment:
Waarom is dit een bekende derde (000000024)?
- minderjarige heeft 1 ouder
- minderjarige is staande huwelijk geboren
- er is gerechtelijke uitspraak 1D

ik denk dat je er niet vanuit mag gaan dat die derde uit de uitspraak 1D dezelfde persoon is als de partner van moeder die al van rechtswege gezag zou hebben gehad.

Reply:
Je hebt gelijk. Er is verwarring ontstaan wat wij verstaan onder _niet ouder_ (derde) vanuit het domein en _niet ouder_ vanuit gezag-api.

Gezag uitspraak bevat nu `OnbekendeDerde`.